sitegen-maven-plugin/src/main/java/io/helidon/build/sitegen/Page.java

```diff
@@ -172,27 +172,32 @@ public Object get(String attr) {
         private static final String KEYWORDS_PROP = "keywords";
         private static final String H1_PROP = "h1";
         private static final String TITLE_PROP = "title";
+        private static final String PARENT_TITLE_PROP = "parentTitle";
```

Comment:
Let's called this h1Prefix all the way, I think it sounds better than parentTitle.

Reply:
h1 is combination of `parentTitle` - `docTitle`, `h1Prefix` doesn't really sound intuitive here

Reply:
then why did you do `h1Prefix = metaData.parentTitle;` below ?

Reply:
my bad, will clean it out

Reply:
Actually, I do prefer h1Prefix over parentTitle ;)

Reply:
as you wish